bundles/org.openhab.binding.iammeter/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="iammeter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="powermeter">
+		<label>Iammeter Power Meter</label>
+		<description>PowerMeter thing for Iammeter Binding</description>
+
+		<channels>
+			<channel id="voltage_a" typeId="voltage_a" />
+			<channel id="current_a" typeId="current_a" />
+			<channel id="power_a" typeId="power_a" />
+			<channel id="importenergy_a" typeId="importenergy_a" />
+			<channel id="exportgrid_a" typeId="exportgrid_a" />
+			<channel id="frequency_a" typeId="frequency_a" />
+			<channel id="pf_a" typeId="pf_a" />
+			<channel id="voltage_b" typeId="voltage_b" />
+			<channel id="current_b" typeId="current_b" />
+			<channel id="power_b" typeId="power_b" />
+			<channel id="importenergy_b" typeId="importenergy_b" />
+			<channel id="exportgrid_b" typeId="exportgrid_b" />
+			<channel id="frequency_b" typeId="frequency_b" />
+			<channel id="pf_b" typeId="pf_b" />
+			<channel id="voltage_c" typeId="voltage_c" />
+			<channel id="current_c" typeId="current_c" />
+			<channel id="power_c" typeId="power_c" />
+			<channel id="importenergy_c" typeId="importenergy_c" />
+			<channel id="exportgrid_c" typeId="exportgrid_c" />
+			<channel id="frequency_c" typeId="frequency_c" />
+			<channel id="pf_c" typeId="pf_c" />
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Device's IP Address</label>
+				<description>IP address for your device</description>
+				<context>network-address</context>
+				<default>127.0.0.1</default>
+			</parameter>
+			<parameter name="port" type="text" required="true" min="1" max="65535">
+				<label>Device's Port</label>
+				<description>Port</description>
+				<default>80</default>
+			</parameter>
+			<parameter name="refreshInterval" type="text" required="false" unit="s">
+				<label>Refresh Interval</label>
+				<default>60</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="voltage_a">
+		<item-type>Number:V</item-type>
```

Comment:
See https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units
```suggestion
		<item-type>Number:ElectricPotential</item-type>
```

Reply:
changed.